Assets/Scripts/CommonEnemy.js

```diff
@@ -54,6 +55,13 @@ private function DropLoot() {
 private function Kill() {
   Debug.Log("Killed enemy (" + gameObject.ToString() + ")");
 
+  // adding points when this is killed
+  try {
```

Comment:
Tää toteutus varmaan riitää tähän hätään. Toisaalta ois nättiä et pelaajalla ois pistecountteri, jota sitten kasvatettais viholiisesta saaduilla pisteillä. Ja pelaajan pisteet sitten päivittyis siihen näyttöön. Tässä nykyisessä on nimittäin se, että jos tota `Kill()` funktioo kutsutaan ilman et pelaaja tappaa vihollisen, niin silloinkin pelaajan pisteet kasvaa